README.md

```diff
@@ -42,7 +42,7 @@ http://pcottle.github.io/learnGitBranching/?gist_level_id=a84407351f9c9f0cb241
 
 ## Reporting Bugs / Opening Issues
 
-When reporting bugs, try running the command `debug_copyTree()` in your JS console when in a state just before reproducing a bug. This can avoid having to copy over all the commands you used to get into a specific state. (I can then use the `importTreeNow` command to get to that exact state)
+When reporting bugs, try running the command `debug_copyTree()` in your JS console when in a state just before reproducing a bug. This can avoid having to copy all the commands you used to get into a specific state. (I can then use the `importTreeNow` command to get to that exact state)
```

Comment:
There's nothing wrong with "copy over" either. I see it as a similar construction to "move over".